app/uk/gov/hmrc/customs/declaration/connectors/BatchFileCustomsNotificationConnector.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.declaration.connectors
+
+import com.google.inject.{Inject, Singleton}
+import play.mvc.Http.HeaderNames._
+import play.mvc.Http.MimeTypes
+import uk.gov.hmrc.customs.api.common.logging.CdsLogger
+import uk.gov.hmrc.customs.declaration.services.{BatchFileCustomsNotification, DeclarationsConfigService}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpException, HttpResponse}
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+@Singleton
+class BatchFileCustomsNotificationConnector @Inject()(http: HttpClient,
```

Comment:
Rightly or wrongly, the naming convention elsewhere for this is `BatchFileUpload` rather than `BatchFile`. Consider changing for consistency.